7thWorkshop/fSettings.cs

```diff
@@ -105,7 +105,17 @@ public partial class fSettings : Form {
         private void bOK_Click(object sender, EventArgs e) {
 
             Sys.Settings.SubscribedUrls = txtSubscriptions.Lines.ToList();
-            if (txtExtraFolders.Lines.ToList().Count > 0 ) Sys.Settings.ExtraFolders = txtExtraFolders.Lines.ToList();
+
+            //Convert Extra Folders textbox to a list for comparison and editing
+            List<string> ExtraFoldersList = txtExtraFolders.Lines.ToList();
+
+            //Make sure that the direct and music folders are always present
+            if (!ExtraFoldersList.Contains("direct", StringComparer.OrdinalIgnoreCase)) ExtraFoldersList.Add("direct");
+            if (!ExtraFoldersList.Contains("music")) ExtraFoldersList.Add("music");
```

Comment:
Here it is missing, please add the `StringComparer.InvariantCultureIgnoreCase` as well